cli/objects/rule.py

```diff
@@ -262,37 +249,34 @@ def _get_jira_additional_labels(
             Optional[list[str]]: A list of strings representing additional labels.
         """
         labels = []
-
-        if "jira_additional_labels" in rule_dict.keys():
-
-            jira_additional_labels = rule_dict["jira_additional_labels"]
-
-            if isinstance(jira_additional_labels, str):
-                labels.append(jira_additional_labels)
-                return labels
-            elif isinstance(jira_additional_labels, list):
-                for label in jira_additional_labels:
-                    if isinstance(label, str):
-                        if " " in label:
-                            self.logger.error(
-                                f'Label "{label}" in rule {rule_dict} contains spaces. Remove spaces and try again.',
-                            )
-                            exit(1)
-                        else:
-                            labels.append(label)
-                    else:
+        jira_additional_labels = rule_dict.get("jira_additional_labels")
+
+        if isinstance(jira_additional_labels, str):
+            labels.append(jira_additional_labels)
+            return labels
+        elif isinstance(jira_additional_labels, list):
+            for label in jira_additional_labels:
+                if isinstance(label, str):
+                    if " " in label:
```

Comment:
why do you need to check if label has space?
Will Jira not accept it? if so can we just strip it?
also this check is not checked if the user pass label as `str`

Reply:
Jira does not accept spaces in the label. I will update the check for `str`